autonomy/cli/analyse.py

```diff
@@ -160,13 +161,61 @@ def docstrings(ctx: Context, update: bool) -> None:
                 click.echo("No update needed.")
 
 
-@analyse_group.command(name="logs")
-@click.argument("file", type=click.Path(file_okay=True, dir_okay=False, exists=True))
-def parse_logs(file: str) -> None:
-    """Parse logs of an agent service."""
+@analyse_group.command("logs")
```

Comment:
Currently this tools supports parsing, storing and querying with basic filters. The database will be created when first time running the command and you can reset it using `--reset-db`.

For now this only has a really simple filter for specifying starting and ending timestamp. 

Log parsing and storing process is fast enough. I ran this locally and it parsed and indexed 4 log files ~=4 MBs and 1000 lines each, it took about 1 sec (Can use thread pool executor in future if required)

Reply:
i wonder if the file size of the db is signficantly less than that of the logs.

Reply:
Yes, that would be a problem when we have log files in GBs.

Reply:
More realistic input are the logs we have from failed tests runs, often ~300MB (see failed e2e actions)

I don't see this tool helping there yet - although this is the type of analysis we have to do daily

Reply:
Right now I've only added support for parsing the logs from deployments, I'll complete this implementation with all the required filters and actions and then add support for parsing the logs from the e2e runs

Reply:
Yes. This PR should not be merged until we have e2e test support for the command as well!